pages/design/test-cases.md

```diff
@@ -0,0 +1,70 @@
+# ACT Test Case Design
+
+The goal of test cases in ACT rules is to allow implementers to verify that their implementation is consistent with the ACT rule. This requires that a rule has test cases for all of the "important" aspects of it. Beyond that, it is up to the rule authors to determine how extensive to make the list of test cases. A list of test cases full of edge cases would ensure greater consistency between implementers, but may result in fewer implementers deciding to adopt the rule.
+
+To strike a good balance in test cases, the following principles should be considered:
+
+1. Test every "condition" in the rule
+1. Test one thing at a time
+1. Ensure consistency with accessibility
+1. Keep test cases small
+1. Test definitions superficially
+1. Avoid tests cases contested by (accessibility) support
```

Comment:
I thought we want to a requirement along the lines of "based on real-world examples" or such?

Reply:
Updating this now to mention "real-world edge cases" in this first section. I think that should resolve all your comments without significantly changing the document.

Reply:
@WilcoFiers you probably forgot to push some commits. I can't find the updated first section. Neither can I find the explanation about the percentages you mention in https://github.com/act-rules/act-rules.github.io/pull/1356/files#r451468720

Reply:
@WilcoFiers this comment has not been addressed.

Reply:
I did, it's on line 3.